java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S5783_java.json

```diff
@@ -0,0 +1,13 @@
+{
+  "title": "Tested checked exception should be expected from one single call",
```

Comment:
I think the title is still cumbersome.
What about "Only one method invocation is expected when testing checked exceptions"